Cargo.toml

```diff
@@ -17,6 +17,7 @@ rust-version = "1.65"
 [features]
 default = []
 bindgen = ["authenticode-parser-sys/bindgen"]
+static_openssl = ["authenticode-parser-sys/static_openssl"]
```

Comment:
I usually see dashes for feature names.
Also, yara-rust has the `openssl-static` feature. I think it would be better
to use this naming as well. Can you change it?